tests/components/denonavr/test_config_flow.py

```diff
@@ -188,7 +188,7 @@ async def test_config_flow_manual_discover_error(hass):
 
 
 async def test_config_flow_manual_host_no_serial(hass):
-    """Test a successful config flow manually initialized by the user with the host specified and an error getting the serial number but backup mac address success."""
+    """Test a successful config flow manually initialized by the user with the host specified and an error getting the serial number."""
```

Comment:
Please break long strings. The docstring should start with a header line that fits on a single line, after that should optionally follow a blank line and then a body. So write a shorter header line and follow up with a body if needed.

PEP8 says that docstrings should be limited to 72 characters width:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Reply:
I updated the docstrings